Source/WebCore/dom/Element.cpp

```diff
@@ -3237,6 +3237,11 @@ static RefPtr<Element> focusDelegateFromShadowHost(ShadowRoot& target)
     return nullptr;
 }
 
+RefPtr<Element> Element::focusDelegate()
```

Comment:
I think we should call this function findFocusDelegate per WebKit's coding guideline:
https://webkit.org/code-style-guidelines/#names-verb

Also, could you add the following URL in a comment for a reference?
https://html.spec.whatwg.org/multipage/interaction.html#focus-delegate